types/p2/index.d.ts

```diff
@@ -27,27 +24,25 @@ declare namespace p2 {
         overlaps(aabb: AABB): boolean;
         containsPoint(point: [number, number]): boolean;
         overlapsRay(ray: Ray): number;
-
     }
 
     export class Broadphase {
+        static AABB: 1;
+        static BOUNDING_CIRCLE: 2;
```

Comment:
are those constants or static properties (of type `number`)? The source uses those as props, but the intent could be to treat those as constants

Reply:
These are static members in the class.
'readonly' is for instance fields.
So I can't change them to 'readonly'.
There are no readonly or constant statics in TypeScript.

Reply:
that's perfectly possible:
```ts
declare class MyClass {
    static readonly myField: number;
}
```
but no worries, that was just question about current implementation and actual usage.